module2/src/com/epam/arraysofarrays/TaskLauncher.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.arraysofarrays;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class TaskLauncher {
+    private static final int ROW = 3;
+    private static final int COLUMN = 2;
+    private static final int MATRIX_ORDER = 6;
+    public static void main(String[] args) throws IOException {
```

Comment:
It is better to add here new lines
```suggestion

    private static final int ROW = 3;
    private static final int COLUMN = 2;
    private static final int MATRIX_ORDER = 6;
    
    public static void main(String[] args) throws IOException {
```